content/pages/education/gi-bill/post-9-11/ch-33-benefit.md

```diff
@@ -3,7 +3,7 @@ title: Check Benefit
 entryname: post-911-gib-status
 layout: page-react.html
 includeBreadcrumbs: true
-in_maintenance: true
+in_maintenance: false
```

Comment:
@djmunoz1 Should this be taken out of maintenance mode when this gets merged in?

Reply:
If so, could we simply remove this metadata? Or is it convenient to keep it here for future use?